config/manager/manager.yaml

```diff
@@ -94,3 +94,20 @@ spec:
       volumes:
       - emptyDir: {}
         name: tmp
+---
```

Comment:
is this needed? Looks a duplicate

Reply:
Verified this is not needed. We already have the Service for addon-controller. Thank you